gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagActionStoreChangeMonitor.java

```diff
@@ -94,6 +99,9 @@ public DagActionStoreChangeMonitor(String topic, Config config, DagActionStore d
     this.flowCatalog = flowCatalog;
     this.orchestrator = orchestrator;
     this.isMultiActiveSchedulerEnabled = isMultiActiveSchedulerEnabled;
+    // instantiating using default ctor; subsequent PR will handle instantiating with multi-args ctor
+//    this.dagTaskStream = new DagTaskStream();
```

Comment:
so this isn't initialized anywhere?  won't we get an NPE when using it?

Reply:
yea we should use guice to bring in it in, see the `DagActionStoreChangeMonitorFactory`